clients/js/src/ChromaClient.ts

```diff
@@ -339,11 +383,305 @@ export class ChromaClient {
    * });
    * ```
    */
-  public async deleteCollection({
-    name,
-  }: DeleteCollectionParams): Promise<void> {
-    return await this.api
-      .deleteCollection(name, this.tenant, this.database, this.api.options)
-      .then(handleSuccess);
+  async deleteCollection({ name }: DeleteCollectionParams): Promise<void> {
+    await this.init();
+
+    await this.api.deleteCollection(
+      name,
+      this.tenant,
+      this.database,
+      this.api.options,
+    );
+  }
+
+  /**
+   * Add items to the collection
+   * @param {Object} params - The parameters for the query.
+   * @param {ID | IDs} [params.ids] - IDs of the items to add.
+   * @param {Embedding | Embeddings} [params.embeddings] - Optional embeddings of the items to add.
+   * @param {Metadata | Metadatas} [params.metadatas] - Optional metadata of the items to add.
+   * @param {Document | Documents} [params.documents] - Optional documents of the items to add.
+   * @returns {Promise<AddResponse>} - The response from the API. True if successful.
+   *
+   * @example
+   * ```typescript
+   * const response = await client.addRecords(collection, {
+   *   ids: ["id1", "id2"],
+   *   embeddings: [[1, 2, 3], [4, 5, 6]],
+   *   metadatas: [{ "key": "value" }, { "key": "value" }],
+   *   documents: ["document1", "document2"]
+   * });
+   * ```
+   */
+  async addRecords(
+    collection: Collection,
+    params: AddRecordsParams,
+  ): Promise<AddResponse> {
+    if (typeof params.ids === "string") {
```

Comment:
NOTE: This check has been reversed in a subsequent stack. Not changing to avoid further PR churn.